Neos.Media/Classes/Domain/Model/AssetCollection.php

```diff
@@ -44,6 +44,22 @@ class AssetCollection
      */
     protected $tags;
 
+    /**
+     * @var AssetCollection
+     * @ORM\ManyToOne(inversedBy="children", cascade={"persist"})
+     * @ORM\JoinColumn(onDelete="SET NULL")
+     * @Flow\Lazy
```

Comment:
I'm not sure if this annotation works as expected, at least the doc comment for it states:

> his is only relevant for anything based on the generic persistence layer of Flow. For Doctrine based persistence this is ignored.

But it probably won't do any harm